### PR TITLE
muharem: retain at rank 2

### PR DESCRIPTION
**Pull request by muharem**

### CODE REVIEW

File: evidence/muharem/0003-muharem-11-2024-retantion-evidence.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+I have completed the implementation work on the pallet for Meta Transactions, and the PR is now ready for review ([polkadot-sdk/6428](https://github.com/paritytech/polkadot-sdk/pull/6428)).
+
+## Voting record
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|I  |90%   |N/A   |   |  |
+|II |80%   |N/A   |   | There was nothing to vote on. |
+|III|70%   |100%  |   | There was nothing to vote on. |
```
Comment (acatangiu) [inline evidence/muharem/0003-muharem-11-2024-retantion-evidence.md:38]:
```suggestion
|III|70%   |100%  |   |  |
```

**Review by acatangiu — APPROVED** [review]

File: evidence/muharem/0003-muharem-11-2024-retantion-evidence.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+I have completed the implementation work on the pallet for Meta Transactions, and the PR is now ready for review ([polkadot-sdk/6428](https://github.com/paritytech/polkadot-sdk/pull/6428)).
+
+## Voting record
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|I  |90%   |N/A   |   |  |
+|II |80%   |N/A   |   | There was nothing to vote on. |
```
Comment (muharem) [inline evidence/muharem/0003-muharem-11-2024-retantion-evidence.md:37]:
```suggestion
|II |80%   |N/A   |   |  |
```

File: evidence/muharem/0003-muharem-11-2024-retantion-evidence.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+I am currently mentoring a new engineer (@davidk-pt) in the ecosystem who is working on the next iteration of Treasury and Bounty pallets, following the plan I outlined in my previous report ([polkadot-sdk/5500](https://github.com/paritytech/polkadot-sdk/issues/5500)). We have completed implementing new indexes for child bounties ([polkadot-sdk/6255](https://github.com/paritytech/polkadot-sdk/pull/6255)) and added a new call to streamline the bounty and curator approval process ([polkadot-sdk/5961](https://github.com/paritytech/polkadot-sdk/pull/5961)). We're also making progress on refactoring the pallets to use new payout mechanics, which will enable proposals in various assets and multi-chain payouts ([polkadot-sdk/6189](https://github.com/paritytech/polkadot-sdk/pull/6189)).
+
+I have completed the implementation work on the pallet for Meta Transactions, and the PR is now ready for review ([polkadot-sdk/6428](https://github.com/paritytech/polkadot-sdk/pull/6428)).
+
+## Voting record
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|I  |90%   |N/A   |   |  |
+|II |80%   |N/A   |   |  |
```
Comment (anaelleltd) [inline evidence/muharem/0003-muharem-11-2024-retantion-evidence.md:37]: Please make sure to fill in the voting record for your rank.
If nothing was available, write "None" or "Nothing to vote on".

Comment (muharem) [inline evidence/muharem/0003-muharem-11-2024-retantion-evidence.md:37]:
```suggestion
|I  |90%   |N/A   |   | None |
|II |80%   |N/A   |   | None |
```

Comment (ggwpez) [reply evidence/muharem/0003-muharem-11-2024-retantion-evidence.md]: https://fellowship-voting.replit.app/

File: evidence/muharem/0003-muharem-11-2024-retantion-evidence.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+I am currently mentoring a new engineer (@davidk-pt) in the ecosystem who is working on the next iteration of Treasury and Bounty pallets, following the plan I outlined in my previous report ([polkadot-sdk/5500](https://github.com/paritytech/polkadot-sdk/issues/5500)). We have completed implementing new indexes for child bounties ([polkadot-sdk/6255](https://github.com/paritytech/polkadot-sdk/pull/6255)) and added a new call to streamline the bounty and curator approval process ([polkadot-sdk/5961](https://github.com/paritytech/polkadot-sdk/pull/5961)). We're also making progress on refactoring the pallets to use new payout mechanics, which will enable proposals in various assets and multi-chain payouts ([polkadot-sdk/6189](https://github.com/paritytech/polkadot-sdk/pull/6189)).
+
+I have completed the implementation work on the pallet for Meta Transactions, and the PR is now ready for review ([polkadot-sdk/6428](https://github.com/paritytech/polkadot-sdk/pull/6428)).
```
Comment (ggwpez) [inline evidence/muharem/0003-muharem-11-2024-retantion-evidence.md:30]: Can you maybe give a bit of reasoning why the things that you are doing are valuable for Polkadot / or in line with the Manifesto?  

I think the Plaza upgrade and meta transactions are good examples that have potentially high value for Polkadot.

Comment (muharem) [reply evidence/muharem/0003-muharem-11-2024-retantion-evidence.md]: updated

**Pull request merged**